ocaml/idl/datamodel.ml

```diff
@@ -1016,6 +1016,8 @@ let _ =
     ~doc:"An HA statefile could not be created, perhaps because no SR with the appropriate capability was found." ();
   error Api_errors.vif_not_in_map [ "vif" ]
     ~doc:"This VIF was not mapped to a destination Network in VM.migrate_send operation" () ;
+  error Api_errors.network_not_found [ "network" ]
+    ~doc:"This network can not be found in the destination Networks in VM.migrate_send operation" () ;
```

Comment:
The words "can not" should be "cannot", but in any case I feel it is better to give errors names and descriptions that allow us to use them in other circumstances in the future: there's no need to limit this to VM.migrate_send.
```
error Api_errors.remote_network_not_found [ "network" ]
  ~doc:"The specified network does not exist on the remote pool." ();
```
or
```
error Api_errors.network_not_found [ "network" ]
  ~doc:"The specified network does not exist." ();
```